docs/src/components/home/Hero.tsx

```diff
@@ -6,7 +6,97 @@ import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
 import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
 import ContentCopyRounded from '@material-ui/icons/ContentCopyRounded';
-import GradientText from 'docs/src/components/GradientText';
+import GradientText from 'docs/src/components/typography/GradientText';
+import PlayerCard from 'docs/src/pages/components/cards/PlayerCard';
+import TaskCard from 'docs/src/pages/components/cards/TaskCard';
+import NotificationCard from 'docs/src/pages/components/cards/NotificationCard';
+import ThemeChip from 'docs/src/pages/components/chips/ThemeChip';
+import ThemeDatePicker from 'docs/src/pages/components/date-picker/ThemeDatePicker';
+import ThemeSlider from 'docs/src/pages/components/slider/ThemeSlider';
+import FolderTable from 'docs/src/pages/components/tables/FolderTable';
+import ThemeTabs from 'docs/src/pages/components/tabs/ThemeTabs';
+import ThemeTimeline from 'docs/src/pages/components/timeline/ThemeTimeline';
+import ViewToggleButton from 'docs/src/pages/components/toggle-button/ViewToggleButton';
+
```

Comment:
Just pointing out that some of the component demos on the Hero are still using PlusJakartaSans. We could standardize them to use IBM Plex Sans.